Firmware/MotorControl/encoder.hpp

```diff
@@ -84,6 +84,10 @@ class Encoder : public ODriveIntf::EncoderIntf {
     int32_t pos_abs_ = 0;
     float spi_error_rate_ = 0.0f;
 
+    float pos_est_rad_ = 0.0f;  // [rad]
+    float vel_est_rad_ = 0.0f;  // [rad]
+    float pos_cpr_rad_ = 0.0f;  // [rad]
```

Comment:
let's rename xxx_estimate -> xxx_estimate_counts, and then rename xxx_est_rad -> xxx_estimate.

While the counts based one is the "core" one that pertains to the encoder, and from that perspective it makes sense that it's the base one; from the user's perspective it should be the radians one that is core and the counts one just for information.

Reply:
Fixed in 7ce896c